src/administration/integrations.md

```diff
@@ -12,5 +12,6 @@ platform integrations
 ![Cli Integrations](/images/cli-integrations.png)
 
 > **note**
-> If you have created your account using the bitbucket or github oAuth Login in order to use the Platform CLI you will need to setup a
-> password which you can do by visiting this page [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password)
+> If you have created your account using the Bitbucket or GitHub oAuth Login,
+in order to use the Platform CLI, you will need to set up a password by visiting
```

Comment:
Let's put a "then" after the comma here.

Also, we favor soft-wrap for the documentation.  No need to hard wrap it.  (It makes it easier to edit when changing something so you don't need to manually reflow all of the text.)